tests/test_goapidiff.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+
+import pytest
+
+from system.plugins.goapidiff.analyzer import GoApiDiff
+
+
+TESTDATA_DIR = 'testdata'
+
+PATH_PAIRS_IDENTICAL_COMMITS = [
```

Comment:
Would be better to provide only (project, commit) pair instead of using predefined path created in ``prepare`` function. So it is more reusable.